packages/blade/src/components/List/_decisions/decisions.md

```diff
@@ -0,0 +1,219 @@
+# List <!-- omit in toc -->
+Lists display a set of related items that are composed of text/links. Each list item begins with a bullet or a number.
+
+<img src="./list-thumbnail.png" width="380" />
+
+- [Design](#design)
+  - [Unordered List](#unordered-list)
+    - [Unordered List with 3 levels of nesting](#unordered-list-with-3-levels-of-nesting)
+    - [Unordered List with custom icons for bullets](#unordered-list-with-custom-icons-for-bullets)
+    - [Unordered List with inline Link in items](#unordered-list-with-inline-link-in-items)
+  - [Ordered List](#ordered-list)
+    - [Ordered List with 3 levels of nesting](#ordered-list-with-3-levels-of-nesting)
+    - [Ordered List with 3 levels of nesting and _filled background_ numbers](#ordered-list-with-3-levels-of-nesting-and-filled-background-numbers)
+    - [Ordered List with inline Link in items](#ordered-list-with-inline-link-in-items)
+- [API](#api)
+  - [List Component](#list-component)
+  - [List.Item Component](#listitem-component)
+  - [List.Item.Link Component](#listitemlink-component)
+  - [Sample Usage](#sample-usage)
+  - [Alternative APIs](#alternative-apis)
+    - [Hierarchial without explicit nested `List` grouping](#hierarchial-without-explicit-nested-list-grouping)
+    - [Hierarchial with a `nested` prop](#hierarchial-with-a-nested-prop)
+    - [Non-Hierarchial with a `level` prop](#non-hierarchial-with-a-level-prop)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+
+## Design
+- [Figma - List](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=23205%3A446859&t=dT849yOxgX5WHgby-4)
+
+### Unordered List
+#### Unordered List with 3 levels of nesting
+
+<img src="./unordered-list.png" width="380" />
+
+#### Unordered List with custom icons for bullets
+
+<img src="./unordered-list-icon.png" width="380" />
+
+#### Unordered List with inline Link in items
+
+<img src="./unordered-list-link.png" width="380" />
+
+### Ordered List
+#### Ordered List with 3 levels of nesting
+
+<img src="./ordered-list.png" width="380" />
+
+#### Ordered List with 3 levels of nesting and _filled background_ numbers
```

Comment:
Yeah, added the title to ensure we mention the use-case. The 3 level nesting is not added on Figma yet but it will be. I'll update the image once its added